support/openharmony/entry/src/main/ets/pages/Index.ets

```diff
@@ -78,57 +86,87 @@ struct Index {
   build() {
     Column() {
       Row() {
-        Button('⇦').backgroundColor(Color.White)
+        Button('+')
+          .backgroundColor(Color.White)
+          .fontColor(Color.Black)
+          .fontWeight(FontWeight.Bolder)
+          .fontSize(22)
+          .width('12%')
+          .onClick((event) => {
+            if (this.tablist.length==0) {
+              this.tablist.push(2);
+            } else {
+              this.tablist.push(this.tablist[this.tablist.length-1]+1);
+            }
+            // yes this is correct as we always have one tab extra
```

Comment:
Why do we have one extra tab?

Reply:
The extra tab is not in the tablist but a separate ArkUI element. This makes the initialization easier.